tests/features/taxonomy.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Taxonomy
+  In order to manage the taxonomy on the website
+  As an administrator
+  I want to be able to create, edit and delete taxonomy
+
+  @api
+  Scenario: Administrator user creates a new vocabulary and adds a new group and terms.
+    Given I am logged in as a user with the 'administrator' role
+    And I create a new vocabulary "vocabulary_test"
+    And I create a new group "tab" named "group_test" in the vocabulary "vocabulary_test"
```

Comment:
Instead of machine names, we should use label as much as possible. 

For instance instead of "tab", I rather use "Vertical tab" instead of "tab" that is meaningful for business understanding. In the function treating the step, the field group label could be converted into its machine name by a helper function.

Other example, for field and field group, implement the same logic as what exist in the "Manage fields" interface where by default, the machine name is automatically defined from the field label .